packages/venia-concept/src/components/Gallery/items.js

```diff
@@ -2,7 +2,8 @@ import React, { Component } from 'react';
 import { arrayOf, number, shape } from 'prop-types';
 import GalleryItem from './item';
 
-const pageSize = 12;
+// Default of 6 to match the default category page size.
+const pageSize = 6;
```

Comment:
We can pull this from the `storeConfig` query when https://github.com/magento/graphql-ce/pull/650 is released! Can you add a TODO reflecting that?